### PR TITLE
feat(upgrade): update guide with the option for NumFOCUS coc

### PR DESCRIPTION
**Pull request by YurelyCamacho**
## Pull Request description
Update of the guide with the inclusion of the option for the numfocus coc, which was missing in the previous PR about this

Solve #142 

## Pull Request checklists

This PR is a:
- [ ] bug-fix
- [ ] new feature
- [X] maintenance

About this PR:
- [ ] it includes tests.
- [ ] the tests are executed on CI.
- [ ] the tests generate log file(s) (path).
- [ ] pre-commit hooks were executed locally.
- [ ] this PR requires a project documentation update.

Author's checklist:
- [ ] I have reviewed the changes and it contains no misspelling.
- [ ] The code is well commented, especially in the parts that contain more complexity.
- [X] New and old tests passed locally.

## Reviewer's Checklist

- [ ] I managed to reproduce the problem locally from the `main` branch
- [ ] I managed to test the new changes locally
- [ ] I confirm that the issues mentioned were fixed/resolved
```

### CODE REVIEW

**Comment by xmnlab** [comment]
Hi @YurelyCamacho 
It shows a bunch of changes unrelated to this PR. Maybe you didn't created your branch on top of the latest changes on upatream/main. I think that you need to rebase your branch on top of the upatream/main

**Comment by YurelyCamacho** [comment]
@xmnlab I have already made  `git rebase upstream/main`

File: docs/guide.md [hunk 1]
```diff
@@ -792,13 +792,15 @@ the *roadmap*.
...
-and choose between two adaptations of well-known codes of conduct accepted by a
-large part of the community: *A Code of Conduct for Open Source Communities by
-Contributors Covenant* and *The Citizen Code of Conduct*.
+A code of conduct is a set of guidelines that outlines the expected
+behavior of individuals participating in a community or organization. It
+typically specifies the types of behavior that are considered acceptable
+and unacceptable, as well as the consequences for violating the code of
+conduct. In SciCookie you can find and choose between three adaptations
+of well-known codes of conduct accepted by a large part of the
+community: *A Code of Conduct for Open Source Communities by
```
Comment (xmnlab) [inline docs/guide.md:801]:
```suggestion
community: A Code of Conduct for Open Source Communities by
```

File: docs/guide.md [hunk 2]
```diff
@@ -792,13 +792,15 @@ the *roadmap*.
...
+Contributors Covenant*,*The Citizen Code of Conduct*, and the Adapted
```
Comment (xmnlab) [inline docs/guide.md:802]:
```suggestion
*Contributors Covenant*,*The Citizen Code of Conduct*, and the *Adapted
```

File: docs/guide.md [hunk 3]
```diff
@@ -792,13 +792,15 @@ the *roadmap*.
...
+NumFOCUS Code of Conduct.
```
Comment (xmnlab) [inline docs/guide.md:803]:
```suggestion
NumFOCUS* Code of Conduct.
```

**Pull request merged**